samples/bash/src/main/resources/changelog.kts

```diff
@@ -27,5 +28,6 @@ changeLog {
                     export | grep " PWD"
                 """.trimIndent()
         }
+        bashAction(File(Main::class.java.getResource("pwet.sh").file))
```

Comment:
I think you should avoid the `.getResource(...).file`, and just use a String instead.
But, that's a minor remark